src/config.rs

```diff
@@ -62,6 +76,45 @@ impl Conf {
         let conf: Conf = toml::de::from_str(&conf)?;
         Ok(conf)
     }
+
+    // write configuration to a file
+    pub fn write(self, path: impl AsRef<Path>) -> Result<(), Error> {
+        let mut file = File::create(path)?;
+        file.write_all(toml::ser::to_string(&self)?.as_bytes())?;
+        Ok(())
+    }
+}
+
+impl Default for Conf {
+    fn default() -> Self {
+        Self {
+            features: Features::default(),
+            params: Parameters::default(),
+            database: Database::default(),
+            client: IrcConfig {
+                nickname: Some("url-bot-rs".to_string()),
+                alt_nicks: Some(vec!["url-bot-rs_".to_string()]),
+                nick_password: Some("".to_string()),
+                username: Some("url-bot-rs".to_string()),
+                realname: Some("url-bot-rs".to_string()),
+                server: Some("chat.freenode.net".to_string()),
+                port: Some(6697),
+                password: Some("".to_string()),
+                use_ssl: Some(true),
+                encoding: Some("UTF-8".to_string()),
+                channels: Some(vec![]),
+                user_info: Some("Feed me URLs.".to_string()),
+                version: Some("0.1".to_string()),
```

Comment:
Perhaps automatically populate the version?

https://stackoverflow.com/questions/27840394/how-can-a-rust-program-access-metadata-from-its-cargo-package

Reply:
...and was planning to do this in a future PR

Reply:
Fixed in 759e814